constants/task_groups.json

```diff
@@ -395,6 +395,9 @@
         "Unverifiable Claim Classification",
         "Word Sense Disambiguation"
     ],
+    "Token Classification": [
+        "Token Classification"
```

Comment:
Let's add this to the `Sequence Tagging` group instead.

Reply:
It is revised as a comment.